service/src/main/java/org/cbioportal/service/util/MutationMapperUtils.java

```diff
@@ -18,18 +9,21 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
+import java.util.*;
+import java.util.function.Function;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
 
 @Component
 public class MutationMapperUtils {
-    
 
     @Autowired
     private GeneService geneService;
 
     public List<StructuralVariant> mapFusionsToStructuralVariants(List<Mutation> fusions,
             Map<String, String> molecularProfileIdReplaceMap, Boolean filterByProteinChange) {
 
-        
         // gene symbols containing "-" is not considered in the pattern
         String pattern = "^([a-zA-Z0-9_.]+)(?:-|_|\\s)([a-zA-Z0-9_.]+)(?:\\s+(\\w+))?$";
```

Comment:
Huh. I've never seen a capturing group within a non capturing group. I didn't even know that worked.